backend/src/models/DatabaseActions.js

```diff
@@ -32,7 +32,15 @@ module.exports = class DatabaseActions {
       throw error;
     }
   }
-
+  async getAccountDetails(userId, depositId) {
+      try {
```

Comment:
```suggestion
    try {
```

check your commit hook again